scripts/noshake.json

```diff
@@ -317,14 +317,15 @@
   "Mathlib.RingTheory.PowerSeries.Basic":
   ["Mathlib.Algebra.CharP.Defs", "Mathlib.Tactic.MoveAdd"],
   "Mathlib.RingTheory.PolynomialAlgebra": ["Mathlib.Data.Matrix.DMatrix"],
+  "Mathlib.RingTheory.Polynomial.Hilbert":
+  ["Mathlib.RingTheory.PowerSeries.WellKnown"],
```

Comment:
> What are these changes about? Should you be shaking the file rather than announcing you're not shaking it?

I'm not sure. After I pushed my work for the first time, the CI was not successful and the error message says that I need to remove some unused imports. But after I removed the "unused" imports, my files stopped compiling anymore. So I ran `lake exe shake --update` as advised, and my PR passed the CI.